.github/workflows/elixir-lint.yml

```diff
@@ -0,0 +1,51 @@
+name: Elixir CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+    branches:
+      - main
+      - develop
+
+jobs:
+  lint:
+
+    name: Check code styles and run tests
+    runs-on: ubuntu-latest
+
+    services:
```

Comment:
For this lint we will not require the database service if we remove the repo from the application as suggested